packages/relay-test-utils/__tests__/__snapshots__/RelayMockPayloadGenerator-test.js.snap

```diff
@@ -331,14 +331,19 @@ fragment TestFragment on User @argumentDefinitions(condition: {type: "Boolean!"}
     "profile_picture": {
       "uri": "<mock-value-for-field-\\"uri\\">"
     },
-    "__typename": "User",
     "actor": {
       "username": "<mock-value-for-field-\\"username\\">",
-      "__typename": "Comment",
-      "body": {
-        "text": "<mock-value-for-field-\\"text\\">"
+      "__typename": "User",
```

Comment:
This looks like an odd change... 🤔

Reply:
I think what’s happening is that we’re able to skip more redundant nodes as a result of this change (exactly what I’d expect). I need to look at this more and try it locally to confirm.